README.md

```diff
@@ -6,99 +6,157 @@
-1. Run any ONNX model
-2. High performance
-3. Cross platform
+**About ONNX Runtime**
+* [Why use ONNX Runtime](#why-use-onnx-runtime)
+* [Design and Key Features](#design-and-key-features)
 
-## Run any ONNX model
+**[Contributions and Feedback](#contribute)**
 
-### Alignment with ONNX Releases
-ONNX Runtime provides comprehensive support of the ONNX spec and can be used to run all models based on ONNX v1.2.1 and higher. See ONNX version release details [here](https://github.com/onnx/onnx/releases).
+**[License](#license)**
+***
+## Installation
+**Quick Start:** The [ONNX-Ecosystem Docker container image](https://github.com/onnx/onnx-docker/tree/master/onnx-ecosystem) is available on Dockerhub and includes ONNX Runtime (CPU, Python), dependencies, tools to convert from various frameworks, and Jupyter notebooks to help get started.
 
-As of May 2019, ONNX Runtime supports ONNX 1.5 (opset10). See [this table](https://github.com/Microsoft/onnxruntime/blob/master/docs/Versioning.md#version-matrix) for details on ONNX Runtime and ONNX versioning compatibility, 
+Additional dockerfiles for some features can be found [here](https://github.com/microsoft/onnxruntime/tree/master/dockerfiles).
 
-### Traditional ML support
-ONNX Runtime fully supports the ONNX-ML profile of the ONNX spec for traditional ML scenarios.
+### System Requirements
+* ONNX Runtime binaries in the CPU packages use OpenMP and depend on the library being available at runtime in the
+system.
+  * For Windows, OpenMP support comes as part of VC runtime. It is also available as redist packages:
+    [vc_redist.x64.exe](https://aka.ms/vs/15/release/vc_redist.x64.exe) and [vc_redist.x86.exe](https://aka.ms/vs/15/release/vc_redist.x86.exe)
+  * For Linux, the system must have the libgomp.so.1 which can be installed using ```apt-get install libgomp1```.
+* The official GPU builds require the CUDA 9.1 and cuDNN 7.1 runtime libraries being installed in the system.
+* Python binaries are compatible with Python 3.5-3.7. See [Python Dev Notes](https://github.com/microsoft/onnxruntime/blob/master/docs/Python_Dev_Notes.md)
+* Certain operators makes use of system locales. At the very least you will need to install English language package and configure en_US.UTF-8 locale.
+  * For Ubuntu install language-pack-en package
+  * Run the following commands:
+  
+    `locale-gen en_US.UTF-8`
+    
+    `update-locale LANG=en_US.UTF-8`
+  * Follow similar procedure to configure other locales on other platforms.
 
-## High Performance
-ONNX Runtime supports both CPU and GPU hardware through a variety of execution providers. With a variety of graph optimizations and accelerators, ONNX Runtime often provides lower latency and higher efficiency compared to other runtimes. This provides faster end-to-end customer experiences and lower costs from improved machine utilization.
+## APIs and Official Builds
```

Comment:
Does it make sense to have APIs and Official Builds below System Requirements?

Reply:
The APIs and Official Builds are one hierarchy above the system requirements, though it doesn't look clear. I moved the system reqs down below the APIs/binaries list.

Reply:
We wrote the section of system requirements to indicate pre-requisites for using the pkgs. However, this didn't quite come out very well as people still ask me these questions and use them without following these pre-requisites. May be we should rename it to pre-requisites or pkg dependencies?